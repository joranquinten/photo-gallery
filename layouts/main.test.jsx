import React from 'react';
import { shallow } from 'enzyme';

import Main from './main';

describe('Main', () => {
  it('should render without crashing', () => {
    expect(shallow(<Main>Children</Main>)).toMatchSnapshot();
  });
});
