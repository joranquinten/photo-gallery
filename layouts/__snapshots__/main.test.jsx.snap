// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render without crashing 1`] = `
<Fragment>
  <Meta />
  <Nav />
  Children
</Fragment>
`;
